hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/filtermanagement/CreateOrUpdateFilterHeader.java

```diff
@@ -80,6 +85,8 @@
     @Autowired
     private UINotification notification;
 
+    private transient Executor executor;
```

Comment:
this field should be autowired instead of using `SpringContextHelper` using `@Autowired` + `@Qualifier`